### ISSUE TITLE
Problem with Google docs

### ISSUE THREAD

**Issue by haruncetinkaya306** [nvaccess/nvda#16567]
### Actual behavior:

Go to Google docs. Try writing something. NVDA speaks the letters you type and delete twice.
### Expected behavior:

NVDA should announce letters written and deleted once.
#### NVDA installed/portable/running from source:
İnstalled.
#### NVDA version:
2024.1
#### Windows version:
Windows 11, 23 H2
#### Does the problem still occur after restarting your computer?
Yes.
#### Have you tried other versions of NVDA? If so, please report their behavior.
No.
#### Does your issue still occur if NVDA add-ons are disabled?
I don't use plugins.
#### Does the problem still occur after running the COM Registry Fix Tool from NVDA's tools menu?
Yes.

**Comment by Adriani90** [comment]
Did you enable braille support as well in google docs accessibility settings or by pressing ctrl+alt+h?

**Comment by haruncetinkaya306** [comment]
@Adriani90
I had the exact problem here and couldn't find the setting you mentioned:
https://docs.google.com/document/d/1ZQAiWo6F_Hg3ud0F5pQ1mazEMsJhVs-Cjwf5eU7tdRA/edit?hl=tr

**Comment by Adriani90** [comment]
Did you press control+alt+h?Von meinem iPhone gesendetAm 18.05.2024 um 10:30 schrieb haruncetinkaya306 ***@***.***>:﻿
@Adriani90
I had the exact problem here and couldn't find the setting you mentioned:
https://docs.google.com/document/d/1ZQAiWo6F_Hg3ud0F5pQ1mazEMsJhVs-Cjwf5eU7tdRA/edit?hl=tr

**Comment by haruncetinkaya306** [comment]
@Adriani90
Yes, enabling Braille support solves the problem, but isn't it still a problem? Because there is no such problem in Narrator.

**Comment by Adriani90** [comment]
Braille support needs to be enabled to make NVDA work properly with Google docs.Von meinem iPhone gesendetAm 18.05.2024 um 12:49 schrieb haruncetinkaya306 ***@***.***>:﻿
@Adriani90
Yes, enabling Braille support solves the problem, but isn't it still a problem? Because there is no such problem in Narrator.

**Comment by Adriani90** [comment]
Closing as works properly.
